gpii/node_modules/flowManager/src/UserLogonStateChange.js

```diff
@@ -38,7 +38,7 @@ gpii.flowManager.userLogonStateChange.errorResponse = function (request, message
 };
 
 gpii.flowManager.userLogonStateChange.startLifecycle = function (lifecycleManager, lifecyclePayload, event) {
-    fluid.log("userLogonStateChange.startLifecycle got final payload " + JSON.stringify(lifecyclePayload, null, 2));
+    gpii.logFully("userLogonStateChange.startLifecycle got final payload ", fluid.censorKeys(lifecyclePayload, ["solutionsRegistryEntries"]));
```

Comment:
why not use the renderMegapayload function here? Is it cause the output from that would still be very long with all the solutionIds listed? It could be misleading for anyone debugging to just censor it... Anyway, it's no big deal if the point is to keep the payloads a reasonable size